HW6/models.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from pydantic import BaseModel, Field
+
+
+class UserCreate(BaseModel):
+    Имя: str = Field(max_length=32)
+    Фамилия: str = Field(max_length=32)
+    email: str = Field(max_length=128)
+    password: str = Field(min_length=3)
+
+
+class UserRead(BaseModel):
+    id: int
+    Имя: str = Field(max_length=32)
+    Фамилия: str = Field(max_length=32)
+    email: str = Field(max_length=128)
+    password: str = Field(min_length=3)
```

Comment:
Модели почти идентичны, значит можно одну наследовать от другой
```suggestion
class UserRead(UserCreate):
    id: int
```